src/perform_step/rkn_perform_step.jl

```diff
@@ -144,27 +144,27 @@ end
     ku = uprev + dt * (c5 * duprev + dt * (a51 * k1 + a52 * k2 + a53 * k3 + a54 * k4))
     kdu = duprev + dt * (abar51 * k1 + abar52 * k2 + abar53 * k3 + abar54 * k4)
 
-    k5 = f.f1(duprev, ku, p, t + dt * c5)
```

Comment:
I missed that in the first implementation https://github.com/SciML/OrdinaryDiffEq.jl/pull/1948, thank you for fixing it!